packages/ra-ui-materialui/src/preferences/Configurable.stories.tsx

```diff
@@ -218,3 +218,8 @@ export const I18n = () => {
 };
 
 export const NotInContext = () => <TextBlock />;
+
+export const usePreferenceNotInConfigurable = () => {
```

Comment:
```suggestion
export const UsePreferenceNotInConfigurable = () => {
```

Not sure it's necessary to create a story for this corner-case though...